Misc/NEWS.d/next/Library/2021-09-04-11-46-05.bpo-21302.zTYR1A.rst

```diff
@@ -0,0 +1 @@
+In Unix operating systems eg: Linux, macOS, FreeBSD etc. time.sleep() has a resolution of nanoseconds with using clock_nanosleep() or nanosleep() function. In Windows, time.sleep() has a resolution of microseconds with using waitable timer from Win32 API.
```

Comment:
The Windosw code path is not modified, so you should not mention it in the *Change* log.

Reply:
Please look at my new Windows implementation of time.sleep(), too. I changed the sleep implementation in Windows, it is based on a Win32 waitable timer object instead of a dummy signal waiting and timeout.